scripts/calculateRevenue/protocols/beefy/types.ts

```diff
@@ -0,0 +1,30 @@
+import { BeefyInvestorTransaction } from '../../../protocol-filters/beefy'
```

Comment:
You can ignore the existing types here, I just pulled them out from the old `beefy.ts` since they're now shared more widely.